listenbrainz/webserver/templates/index/terms-of-service.html

```diff
@@ -0,0 +1,37 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Terms of Service - ListenBrainz{%- endblock -%}
+{%- block content -%}
+<h2 class="page-title">Terms of Service</h2>
+
+<p>As one of the projects of the <a href="https://metabrainz.org/projects">MetaBrainz Foundation</a>,
```

Comment:
As one of the projects of the <a href="https://metabrainz.org/projects">MetaBrainz Foundation</a>,
ListenBrainz' terms of service are defined by our social contract and our privacy policies of the Foundation.
You will find these detailed on the MetaBrainz website

I find this to be a bit more clear